src/main/java/ru/sbtqa/tag/pagefactory/stepdefs/GenericStepDefs.java

```diff
@@ -83,6 +85,7 @@ public void userActionInBlockNoParams(String block, String action) throws PageIn
     @And("ru.sbtqa.tag.pagefactory.userActionInBlockTableParam")
     public void userActionInBlockTableParam(String block, String action, DataTable dataTable) throws PageInitializationException, NoSuchMethodException {
         PageFactory.getInstance().getCurrentPage().executeMethodByTitleInBlock(block, action, dataTable);
+        ParamsHelper.addAttachment(dataTable.toString().getBytes(), "DataTable", Type.TEXT);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "DataTable" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1192)